lib/index.ts

```diff
@@ -61,15 +61,23 @@ async function setup(devicePath: string): Promise<express.Application> {
 	app.post(
 		'/dut/flash',
 		async (req: express.Request, res: express.Response) => {
+			res.writeHead(202, {
+				'Content-Type': 'text/event-stream',
+				Connection: 'keep-alive',
+			});
+
 			const timer = setInterval(() => {
 				res.write('Still Flashing');
 			}, httpServer.keepAliveTimeout);
 
 			try {
-				res.status(202);
+				worker.on('progress', progress => {
+					res.write(`progress: ${JSON.stringify(progress)}`);
```

Comment:
Who will be consuming this?
Maybe
```typescript 
res.write(JSON.stringify({ progress }));
```
is better?

Reply:
The client will be consuming this,